frontend/src/components/commentSection/CommentSection.jsx

```diff
@@ -255,10 +255,18 @@ const CommentSection = ({loggedInAuthor, commentsId, commentCount, postAuthorId,
             }
 
             if (postHostName === "backend-404.herokuapp.com"){
-                try {
-                    
-                    await axios.post(commentsId, newComment, {
-                        headers: {
+              // TEAM 4 says posting comments to their backend is not possible
+              // Thus this post to commentsId is useless
+              try {
+                var urlArray = commentsId.split("/");
+                var postId = urlArray[6]; // Get the post UUID
```

Comment:
`http://backend-404.herokuapp.com/authors/{AUTHOR_UUID}/posts/{POST_UUID}` when split equals the following indexes:
0 - "http"
1 - ""
2 - "backend-404.herokuapp.com"
3 - "authors"
4 - "AUTHOR_UUID"
5 - "posts"
6 - "POST_UUID"